WrightTools/data/_join.py

```diff
@@ -88,6 +88,19 @@ def join(
     for d in datas[1:]:
         if d.axis_expressions != axis_expressions:
             raise wt_exceptions.ValueError("Joined data must have same axis_expressions")
+        for a in d.axes:
+            if len(np.squeeze(a.variables[0][:]).shape) > 1:
+                raise wt_exceptions.ValueError(
+                    "Axis '"
+                    + a.natural_name
+                    + "' of datas["
+                    + str(datas.index(d))
+                    + "] ('"
+                    + d.natural_name
+                    + "') must be 1D, but currently has "
+                    + str(len(np.squeeze(a.variables[0][:]).shape))
+                    + " nontrivial axes"
```

Comment:
consider using an "f-string"--they are very convenient!  e.g.
```
f"Axis {a.natural_name} of {d.natural_name} must be 1D, but currently has..."
```

Reply:
Looks more intuitive for future editors. I'll convert.